db/migrate/20220603110850_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Kindly reference author instead of user

```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Hello @Cathella, please the user is the same as the author

Reply:
Thanks a lot @Cathella for drawing my attention to this. The issue has been resolved